### PR TITLE
add NixOS/nixos-metrics metrics via VictoriaMetrics

### PR DESCRIPTION
**Pull request by Radvendii**
The metrics we've been collecting at github:NixOS/nixos-metrics are currently viewable at https://nixos.github.io/nixos-metrics/. It would be nice to get them into the community Grafana, so people will know where to find them and so they can be accessed using all of Grafana's fancy features (data analysis, comparison to other metrics, etc).

We are aiming to use VictoriaMetrics rather than Prometheus because
1. It allows us to add more metrics downstream that will automatically get picked up without making a PR here every time
2. It allows back-filling the data we've already collected.

I'm not sure if there's an easy way to allow external yet secure access directly to the VictoriaMetrics instance. So instead I've opted for the server to *pull* data and push it to VictoriaMetrics internally.

**NOTE:** There is one manual step required. VictoriaMetrics has to be added as a source in Grafana of type Prometheus and with the address `http://localhost:8428/prometheus`

### CODE REVIEW

File: delft/eris.nix [hunk 1]
```diff
@@ -590,6 +590,28 @@ in
       (import ../channels.nix).channels;
   };
 
+  # pull nixos metrics from github:NixOS/nixos-metrics to local VictoriaMetrics
+  # every day at 09.00
+  services.cron = {
```
Comment (edolstra) [inline delft/eris.nix:595]: This should be a systemd timer. Cron is a pain in the ass to use/test.

Comment (Radvendii) [reply delft/eris.nix]: Fixed.

**Comment by thufschmitt** [comment]
@grahamc : Eelco told me you've deployed this but I can't find these metrics anywhere on https://monitoring.nixos.org/grafana . Any pointer as to where this is?

**Comment by mweinelt** [comment]
@thufschmitt The data source was never added to Grafana.

**Pull request merged**